homeassistant/components/shelly/const.py

```diff
@@ -265,7 +265,15 @@ class BLEScannerMode(StrEnum):
 
 CONF_GEN = "gen"
 
-VIRTUAL_COMPONENTS = ("boolean", "button", "enum", "input", "number", "text")
+VIRTUAL_COMPONENTS = (
```

Comment:
Isn't better if we load the bar from aioshelly so we are 100% they are always in sync ?